client/components/my-plan-card/README.md

```diff
@@ -34,9 +34,10 @@ export default class extends React.Component {
 
 The following props can be passed to the My Plan Card component:
 
-* `buttonLabel`: ( string ) Action button label
-* `buttonTarget`: ( string ) Action button target (`href`)
-* `expirationDate`: ( string ) Plan or product expiration date, e.g. `2019-11-14T16:53:25+00:00 ` (ISO 8601)
+* `action`: ( element | node ) Action button element or node.
+* `isExpiring`: ( bool ) Flag indicating that the plan or product will expire soon
+* `isPlaceholder`: ( bool ) Flag indicating that the component in is a loading state
```

Comment:
Is this implemented yet in, Can you add a example of this state?

Reply:
So the loading state was not implemented yet, but I decided to add a support for this now. Please have a look at [the most recent commit](https://github.com/Automattic/wp-calypso/pull/37637/commits/d6172ef0e745dfa7efdd568945607623d0848ae5).

Here's a screenshot of how this looks:
![Screenshot 2019-11-15 at 17 49 36](https://user-images.githubusercontent.com/478735/68960374-55a9ba00-07d0-11ea-9fb9-67862059345d.png)